src/js/components/mapWidgets/widgetContent/penContent.tsx

```diff
@@ -1,9 +1,21 @@
-import React, { FunctionComponent } from 'react';
-import { useSelector } from 'react-redux';
+import React, { FunctionComponent, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
```

Comment:
Can we use relative imports for most things? eg `from images/penIcon.svg` `from js/store/appState...` etc etc